docs/docs/release-notes/0.1.2.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: doc-page
+title: 0.1.2 release notes
+---
+
+0.1.2 will be the first public release of Dotty.
+Being first, those notes are incomplete.
+This document is work in progress until 0.1.2-final is released.
+This note is up-to-date based on 0.1.2-RC1 and will be updated 
+with changes that happen during RC stabilization cycle.
+
+# Required Java Version
+  
+Dotty 0.1.2 targets Java 8. We don't have plans so far to add support for earlier versions of Java. 
+
+# IDE support for Dotty
+
+# Reporting Bugs / Known Issues
+
+Please [file](https://github.com/lampepfl/dotty/issues) any bugs you encounter. If you’re unsure whether something is a bug, 
+please ask on Dotty [gitter channel](https://github.com/lampepfl/dotty).
+
+# SBT builds dotty that builds dotty
+
+
+# Dotty Doc
```

Comment:
Is that the official spelling? Because there's been JavaDoc, Scaladoc, and now Dotty Doc.